solution/backend/regulations/templates/regulations/homepage.html

```diff
@@ -25,11 +18,12 @@
 <main>
     <nav id="placeholderNav"></nav>
     <left-nav-collapse contents-description="table of contents">
-        <toc-container
-            slot="nav-contents"
-            api-url="{{ API_BASE }}"
-            home-url="{% url 'homepage' %}"
-        ></toc-container>
+        <template #nav-contents>
+            <toc-container
+                api-url="{{ API_BASE }}"
+                home-url="{% url 'homepage' %}"
+            ></toc-container>
+        </template>
```

Comment:
Named slots must be used on a `<template>` element.  See https://vuejs.org/guide/components/slots.html#named-slots